setup.cfg

```diff
@@ -22,18 +22,19 @@ packages = find:
 include_package_data = True
 install_requires =
   appdirs>=1.4
-  asdf>=2.9.1
-  asdf-astropy
+  asdf>=2.9.2
+  asdf-astropy>=0.1.1
   asdf-transform-schemas
   astropy>=5
-  dask[array]>=2
+  dask[array]>=2021.8.0
   globus-sdk>=3.0
   gwcs>=0.18.0
-  matplotlib>=3.4
+  matplotlib>=3.5
   ndcube[plotting,reproject]>=2.0
-  numpy>=1.21
-  parfive[ftp]>=1.3
-  sunpy[net,asdf]>=4
+  numpy>=1.22
+  parfive[ftp]>=1.5
+  sunpy[net,asdf]>=4.0.7
+  setuptools>=59 # Minimum version older than 60 to avoid distutils warnings
```

Comment:
aren't this line and comment counter to each other?

Reply:
>=

:shrug: Going from the previous comment, I don't actually know which versions are the ones giving the warnings

Reply:
```suggestion
  setuptools>=59
```